scripts/start.js

```diff
@@ -119,35 +151,63 @@ compiler.plugin('done', function (stats) {
     console.log('Use ' + chalk.yellow('// eslint-disable-next-line') + ' to ignore the next line.');
     console.log('Use ' + chalk.yellow('/* eslint-disable */') + ' to ignore all warnings in a file.');
   }
-});
+}
 
-function openBrowser() {
-  if (process.platform === 'darwin') {
-    try {
-      // Try our best to reuse existing tab
-      // on OS X Google Chrome with AppleScript
-      execSync('ps cax | grep "Google Chrome"');
-      execSync(
-        'osascript ' +
-        path.resolve(__dirname, './openChrome.applescript') +
-        ' http://localhost:3000/'
-      );
-      return;
-    } catch (err) {
-      // Ignore errors.
-    }
-  }
-  // Fallback to opn
-  // (It will always open new tab)
-  opn('http://localhost:3000/');
+function exponentialBackoff(step) {
+  return Math.pow(2, step);
 }
 
-new WebpackDevServer(compiler, {
+const clientCompiler = webpack(configClient, handleCompileClient);
+clientCompiler.plugin('invalid', webpackOnInvalid);
+clientCompiler.plugin('done', webpackOnDone);
+
+// Here's the server compiler
+webpack(configServer, function(error, stats) {
```

Comment:
So here's the idea, server is being automatically watched, re-compiled & restarted.